tests/multidep_modules/eslint-4.19.1/package-lock.json

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
do we need to commit the `package-lock.json` file? seems strange when the main package is using `yarn.lock` 🤔